tuf/roledb.py

```diff
@@ -124,6 +124,8 @@ def create_roledb_from_root_metadata(root_metadata, repository_name='default'):
     if rolename == 'root':
       roleinfo['version'] = root_metadata['version']
       roleinfo['expires'] = root_metadata['expires']
+      roleinfo['previous_keyids'] = root_metadata.get('signing_keyids', [])
+      roleinfo['previous_threshold'] = root_metadata.get('threshold', 1)
```

Comment:
`root_metadata` doesn't have a 'signing_keyids' or `threshold` entry.  I think `roleinfo.get('keyids')` and `roleinfo.get('threshold')` are needed here instead.

See https://github.com/theupdateframework/tuf/pull/341/files#diff-43de4bd3342179b5413e1971566ed4cfR123
and https://github.com/theupdateframework/tuf/blob/develop/tests/repository_data/repository/metadata/root.json#L58-L82